src/annotator/adder.js

```diff
@@ -252,19 +174,8 @@ export class Adder {
    * @param {number} top - Vertical offset from top edge of viewport.
    */
   showAt(left, top, arrowDirection) {
-    this.element.className = classnames({
-      'annotator-adder': true,
-      'annotator-adder--arrow-down': arrowDirection === ARROW_POINTING_DOWN,
-      'annotator-adder--arrow-up': arrowDirection === ARROW_POINTING_UP,
-    });
-
-    // Some sites make big assumptions about interactive
-    // elements on the page. Some want to hide interactive elements
-    // after use. So we need to make sure the button stays displayed
-    // the way it was originally displayed - without the inline styles
-    // See: https://github.com/hypothesis/client/issues/137
-    this.element.querySelector(ANNOTATE_BTN_SELECTOR).style.display = '';
-    this.element.querySelector(HIGHLIGHT_BTN_SELECTOR).style.display = '';
+    // TODO - Check that https://github.com/hypothesis/client/issues/137 remains
+    // fixed.
```

Comment:
This TODO is the main thing I need to check before removing the WIP label.

Reply:
Yes. The linked issue remains fixed. The website has quite possibly changed since the issue was filed, but I think based on [this comment](https://github.com/hypothesis/client/issues/137#issuecomment-249703083) that use of Shadow DOM would also have resolved it, as it seems the website was modifying inline style properties of the toolbar buttons. As well as protecting us from the page's CSS, Shadow DOM also hides elements from a naive `querySelectorAll(...)`.